sos/plugins/openvswitch.py

```diff
@@ -13,10 +13,12 @@
 from os.path import join as path_join
 from os import environ
 
+import re
+
 
 class OpenVSwitch(Plugin):
-    """ OpenVSwitch networking
-    """
+
+    short_desc = 'OpenVSwitch networking'
```

Comment:
The change to `short_desc` is a sos-4.0 thing, the legacy branch still uses the class docstrings.